### PR TITLE
qlandkartegt: remove

### PR DESCRIPTION
**Pull request by totoroot**
###### Description of changes

Remove `qlandkartegt` since it was marked as insecure.
See https://github.com/NixOS/nixpkgs/pull/201028

@sikmir Please approve removal

QLandkarte GT hasn't seen any updates since 2015 and can therefore probably be considered unmaintained/abandoned:
https://sourceforge.net/projects/qlandkartegt/

On the AUR (patches fetched from there) it has been unmaintained as well:
https://aur.archlinux.org/packages/qlandkartegt

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — APPROVED** [review]
give me a ping if this starts to get stale

**Comment by sikmir** [comment]
That's a bit sad to remove, since qlandkartegt just works although not being updated since 2015. I still use it from time to time. Yes, I know there are alternatives, but... qlandkartegt does its job well. I've just moved it to my NUR repo (https://github.com/sikmir/nur-packages/commit/a8d1fc3538abcd114d5b3c55ea63e95fa5ab9acb), so it should be available at `nur.repos.sikmir.qlandkartegt` for those who still needs it.

**Review by sikmir — APPROVED** [review]

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -536,6 +536,7 @@ mapAliases ({
   g4py = python3Packages.geant4; # Added 2020-06-06
   gaia = throw "gaia has been removed because it seems abandoned upstream and uses no longer supported dependencies"; # Added 2020-06-06
   gammy = throw "'gammy' is deprecated upstream and has been replaced by 'gummy'"; # Added 2022-09-03
+  garmindev = "'garmindev' has been removed from nixpkgs, as it was broken and unmaintained"; # Added 2023-04-17
```
Comment (NickCao) [inline pkgs/top-level/aliases.nix:539]: Missing a `throw` there.

Comment (totoroot) [reply pkgs/top-level/aliases.nix]: Thanks, fixed!

**Comment by totoroot** [comment]
> give me a ping if this starts to get stale

@SuperSandro2000 Can this get merged now?

**Pull request merged**